demos/shadow/shadow_demo_main/demo_config.h

```diff
@@ -117,40 +117,46 @@
  *
  * No two clients may use the same client identifier simultaneously.
  */
-#define CLIENT_IDENTIFIER      "testclient"
+#define CLIENT_IDENTIFIER         "testclient"
 
 /**
  * @brief Size of the network buffer for MQTT packets.
  */
-#define NETWORK_BUFFER_SIZE    ( 1024U )
+#define NETWORK_BUFFER_SIZE       ( 1024U )
 
 /**
  * @brief The name of the operating system or SDK that the application is
```

Comment:
can this be the SDK name as well? If not, I think a comment update is required to remove 'or SDK'

Reply:
I think I will take out the "or SDK" to be more specific.